Sources/AblyAssetTrackingCore/Model/Resolution.swift

```diff
@@ -20,6 +20,10 @@ public struct Resolution: Codable, CustomDebugStringConvertible {
      be filtered out.
      Used to govern the frequency of updates requested from the underlying location provider, as well as the frequency
      of messages broadcast to subscribers.
+     
+     Waning:
+     Change of this value has no effect in `CoreLocation` engine, for example, when  Publisher `Resolution` is set as constant.
```

Comment:
It means that change of the `desiredInterval` property here does nothing in the CoreLocation engine because there is no option to change something like "desired interval" or "interval" in CLLocationManager. I've placed the link to CLLocationManager documentation as a proof.
The second part tells us if we used "constant location resolution" which uses "Resolution" object, the change of this property has no effect.
But the "Resolution" object can be also in the "DefaultResolutionPolicy" which resolving "Resolution" for every "trackable" object. Here, the property "desiredInterval" is in use.

Reply:
TBH I also think that it's a bit vogue description of said Warning. Also Warning vs Waning

Reply:
Yeah, I'm quite confused by this. From the existing comment line above this one:

> Used to govern the frequency of updates requested from the underlying location provider, as well as the frequency of messages broadcast to subscribers.

Let's split this in two:

> used to govern the frequency of updates requested from the underlying location provider

If I'm understanding correctly what you've written above, you're saying that, in fact, it is not possible to "govern the frequency of updates requested from the underlying location provider" with Core Location? In that case, do we need to modify the original code comment?

As for the link to the Core Location documentation, I think that it confuses more than it helps, at the moment.

> as well as the frequency of messages broadcast to subscribers

I'm not super familiar with the library so perhaps I'm not wording the question well, but when the user sets the publisher’s `constantLocationEngineResolution` property, will the `desiredInterval` property still be used to control the "frequency of messages broadcast to subscribers"?

Reply:
Changed here 7e09e9f0e9a4af58b253431bece7286d0429cf86
> I'm not super familiar with the library so perhaps I'm not wording the question well, but when the user sets the publisher’s constantLocationEngineResolution property, will the desiredInterval property still be used to control the "frequency of messages broadcast to subscribers"?

I guess you can find the answer to this question in the discussion with Kacper.

Reply:
> > I'm not super familiar with the library so perhaps I'm not wording the question well, but when the user sets the publisher’s constantLocationEngineResolution property, will the desiredInterval property still be used to control the "frequency of messages broadcast to subscribers"?
> 
> I guess you can find the answer to this question in the discussion with Kacper.

My understanding is that, even if a `constantLocationEngineResolution` is set, its `desiredInterval` has _no effect_ on deciding whether to send a message to a subscriber - the `ResolutionConstraints` / associated remote requests of the `Trackable`s are used, as normal.

That means that the comment "as well as the frequency of messages broadcast to subscribers" needs to be qualified somewhere, to explain that it does not apply to the `desiredInterval` of the `constantLocationEngineResolution`. Perhaps a comment on the `constantLocationEngineResolution` property itself?

Reply:
> If I'm understanding correctly what you've written above, you're saying that, in fact, it is not possible to "govern the frequency of updates requested from the underlying location provider" with Core Location? In that case, do we need to modify the original code comment?

Did you have thoughts about this one, @lukasz-szyszkowski? I think to remove that sentence from the comment, because, as you've pointed out, it's not true on iOS. (We might want to also highlight the fact that this behaviour differs from Android.)

Reply:
I removed it here 2ba81b6920de723192185d0554f7a6ab6374d447

Reply:
Closing this conversation in favour of moving this PR forward. @lawrence-forooghian I will create a new issue from your comment (documentation about difference from Android)